src/commands/Utility/language.ts

```diff
@@ -148,6 +153,28 @@ const command: Command = {
 					})
 				}
 			})
+		} else if (subCommand === "reset") {
+			const result = await collection.updateOne({ id: interaction.user.id }, { $unset: { lang: true } })
+			if (result.modifiedCount) {
+				randomTip = generateTip(getString, language)
```

Comment:
and language is...?

Reply:
string | undefined, i assumed it was fine to leave it as that because generateTip uses getString which defaults to en if language is undefined. Maybe I'm missing something 🤔

Reply:
language is the language option used for commands where that option exists, which is not the case

Reply:
oooooh yeah, you're right, of course